### ISSUE TITLE
Torch damage

### ISSUE THREAD

**Issue by daretmavi** [minetest-mods/3d_armor#22]
Hi,

if I activate armor_fire_protect = true then Torch damage is activated as well.

Could you add a switch to just turn the torch damage off, even is fire protect is activated?

**Comment by BuckarooBanzay** [comment]
> Could you add a switch to just turn the torch damage off, even is fire protect is activated?

Sounds like a nice-to-have feature :+1: 

Relevant parts (just from glancing over the code):

https://github.com/minetest-mods/3d_armor/blob/7436fcdffedf5e37fb11370c3cd544afb4cae539/3d_armor/init.lua#L424-L433

**Comment by sirrobzeroone** [comment]
How about this for a solution?

Instead of adjusting the fire.protect function, I set any item/node that is in the armor.fire_nodes list which contains the word torch to nil? effectivly removing all torches from the list. I did a find incase someone decides to register other torches in the global "Armor_fire_nodes"  which are also torches.

**to settingtypes.txt line 47 I added**
```
# Enable fire damage from torches (defaults true if using ethereal mod).
armor_fire_protect_torch (Enable fire protection torch damage) bool false
```

**to api.lua I added below line 99**
```
fire_protect_torch = minetest.get_modpath("ethereal") ~= nil,
```

**to init.lua just below Load config and above Mod Compatability ~= line 65 I added:**
```
-- Remove torch damage if fire_protect_torch == false
if armor.config.fire_protect_torch == false and armor.config.fire_protect == true then
	for k,v in pairs(armor.fire_nodes) do
		for k2,v2 in pairs(v) do
			if string.find (v2,"torch") then
				armor.fire_nodes[k] = nil
			end
		end
	end
end
```

I did some testing and couldnt detect any issues with the above and it seems to work well, still burnt on fires and not burnt on torches when disabled.

**Comment by BuckarooBanzay** [comment]
fixed by #29 (thanks @sirrobzeroone)
